pages/index.vue

```diff
@@ -28,25 +28,7 @@
 
       <!-- Filter  -->
       <div class="col-span-6 lg:col-span-2 lg:col-end-13">
-        <!-- Insert filter component here  -->
-        <!-- @filter-by-region="filterByRegion"  -->
-
-        <!-- <Filters
-          :region="region"
-          :regions="regions"
-          @change="filterByRegion(region)"
-        /> -->
-
-        <select
-          class="bg-white dark:bg-slate shadow rounded w-full"
-          v-model="region"
-          @change="filterByRegion(region)"
-        >
-          <option selected>Filter By Region</option>
-          <option v-for="option in regions" :key="option" :value="option">
-            {{ option }}
-          </option>
-        </select>
+        <Filters :list="regions" v-model="region"/>
```

Comment:
There's a missing click event method on this line. No?

Reply:
nope